osc-language-configuration.json

```
@@ -26,5 +26,11 @@
     "indentationRules": {
         "increaseIndentPattern": "{(init|frame|frame_ai|if|else|(macro|trigger):([\\w\\-+]+))}",
         "decreaseIndentPattern": "{(end|endif)}"
+    },
+    // Code folding
+    "folding": {
+    "markers": {
+      "start": "{((macro|trigger):([\\w\\-+]+))|if|else}",
+      "end": "{(endif|end)}"
```

There's a missing closing bracket here, and imo the indentation should be consistent with the rest of the file.
```suggestion
        "markers": {
            "start": "{((macro|trigger):([\\w\\-+]+))|if|else}",
            "end": "{(endif|end)}"
      }
```